framework/resource/manager.go

```diff
@@ -305,6 +305,83 @@ func (m *Manager) DestroyAll(args ...interface{}) error {
 	return result.Err()
 }
 
+// StatusAll invokes the GetStatus method on all the resources under management.
```

Comment:
It doesn't look like this actually invokes GetStatus for each resource

Reply:
True! That's a typo from a careless find/replace. I updated the comment in 2433ce5